_episodes/02-concepts.md

```diff
@@ -62,34 +77,63 @@ preferably machine accessible.
 > one shares with colleagues and collaborators. However, such 
 > information requires significant human resources to interpret and 
 > translate into code. An alternate approach is to encode the metadata
-> using structured markup (RDF, JSON, XML). Often such markup can be 
+> using structured markup (e.g., RDF, JSON, XML). Often such markup can be 
 > standardized to provide machine accessibility.
 {: .callout}
 
-In this example the data and metdata are stored in a [google spreadsheet](https://docs.google.com/spreadsheets/d/11an55u9t2TAf0EV2pHN0vOd8Ww2Gie-tHp9xGULh_dA).
-**TODO** The column headers are described in detail in a JSON document 
-using JSONLD a format that supports semantic annotation. The annotation 
-provides information about the data contained in the column and allows 
-harmonizing the information with other similar tables. The phenotypic 
-information are stored as literals. The imaging data are stored as 
-pointers to files in the NITRC XNAT repository. For example, the JSONLD 
-metadata key tells us that the URLs correspond to anatomical T1-weighted 
+In this example the data and metdata are stored in a 
+[google spreadsheet](https://docs.google.com/spreadsheets/d/11an55u9t2TAf0EV2pHN0vOd8Ww2Gie-tHp9xGULh_dA).
+The phenotypic information are stored as literals. The imaging data are stored as 
+pointers to files in the NITRC XNAT repository. However, this particular example 
+does not have any semantic or type information associated with the input file.
+
+The column headers can be described in detail in a JSON document 
+using [JSONLD](https://json-ld.org/) a format that supports semantic annotation. 
+The annotation provides information about the data contained in the column and 
```

Comment:
Do we need an example JSON-LD document for this simple workflow as to demonstrate how metadata could be added?